### DISCUSSION TITLE
Correct way to have development setup with continuous delivery

### DISCUSSION THREAD

**Discussion by shahidcodes** [pocketbase/pocketbase#3748] [Q&A] [answered]
I’ve a fairly small app. I’ve deployed first version of it and users are loving it. I’ve made several changes to the app locally. Now I want to deploy it to production 

1. How do I make sure I run the migration properly? I can’t find any guide for this, there’s a guide to generate migrations automatically but I found no migrations have been generated locally. Should I try collection dump migration command to generate another migration from local db then apply it to production?
2. Going forward how do I plan so that my migrations are done properly and can be shared via git?
3. How do I integrate all this to continuous delivery ?

**Comment by ganigeorgiev** [comment]
If you use the prebuilt executable, [automigrate](https://pocketbase.io/docs/js-migrations/#automigrate) is enabled by default.
This means that in order to "transfer" you local schema changes to a remote instance, it is enough to copy the `pb_migrations` directory and restart the application (_unapplied migrations are automatically applied on `serve`_).
You can safely commit in git the `pb_migrations` directory to share it with others.

**Reply by ganigeorgiev** [reply, marked as answer]
If you use PocketBase as Go framework, aka. you have your own Go executable, then the migrations will be included in the resulting binary (to setup migrations with Go you could check the [Go Migrations guide](https://pocketbase.io/docs/go-migrations/)).
